model-optimizer/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2018-2020 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# install
+ie_cpack_add_component(model_optimizer REQUIRED)
+
+configure_file(
+    "${CMAKE_CURRENT_SOURCE_DIR}/automation/version.txt.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/version.txt"
+    @ONLY)
+
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
+        DESTINATION deployment_tools/model_optimizer
+        USE_SOURCE_PERMISSIONS
```

Comment:
Why do we need this line? I am not sure that we need to save source permissions.

Reply:
@ilyachur , this is my request. We need this line, because permissions of the MO files in the git are correct: some specific files have execution bit but others don't. Moreover, the old version of the script which was used to generate the MO package used the same logic.

Reply:
Ok, I just thought that we don't need to have execution bit for python scripts, but if we have some tools which require this bit, I am ok. Thank you for clarification.